plenum/test/consensus/view_change/test_view_change_service.py

```diff
@@ -15,10 +15,16 @@
     create_new_view, create_view_change, create_new_view_from_vc, create_view_change_acks, create_batches
 
 
+DEFAULT_STABLE_CHKP = 10
+
+
 @pytest.fixture
-def view_change_service_builder(consensus_data, timer, internal_bus, external_bus, stasher):
+def view_change_service_builder(consensus_data, timer, internal_bus, external_bus, stasher, initial_view_no):
     def _service(name):
         data = consensus_data(name)
+        digest = cp_digest(DEFAULT_STABLE_CHKP)
+        cp = Checkpoint(instId=0, viewNo=initial_view_no, seqNoStart=0, seqNoEnd=DEFAULT_STABLE_CHKP, digest=digest)
+        data.checkpoints.append(cp)
```

Comment:
Why do we just add new checkpoint, but not update stable checkpoint? Was it intended?